<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>moveposotionwindow</class>
 <widget class="QWidget" name="moveposotionwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="info_lb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>561</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Введіть інформацію про переведення</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="cadr_lb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Працівник</string>
   </property>
  </widget>
  <widget class="QListWidget" name="cadr_list">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="position_lb">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>На яку посаду :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="position_cb">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="move_button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Перевести</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cansel_button">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>390</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Скасувати операцію</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
